src/main/resources/data/computercraft/lua/rom/apis/window.lua

```diff
@@ -235,7 +235,17 @@ function create(parent, nX, nY, nWidth, nHeight, bStartVisible)
     -- @type Window
     -- @see term.Redirect
     local window = {}
-
+    function window.isVisible()
+        return bVisible
+    end
+    
+    function window.setVisible(b)
```

Comment:
There's already a `setVisible` function on line 467. It's probably worth moving `isVisible` to there too.

https://github.com/SquidDev-CC/CC-Tweaked/blob/61fb4caaad51b83ac4a8a5999e50d439f94c71bf/src/main/resources/data/computercraft/lua/rom/apis/window.lua#L467